zio-http/shared/src/main/scala/zio/http/ServerSentEvent.scala

```diff
@@ -81,7 +82,8 @@ final case class ServerSentEvent[T](
       sb.append('\n')
     }
     dataLines.foreach { line =>
-      sb.append("data: ").append(line).append('\n')
+      if (isComment) sb.append(line).append('\n')
```

Comment:
I think this is wrong. If you do this, you would not send the comment over the wire and that was the whole point of this or not?